opentelemetry-sdk/tests/metrics/test_periodic_exporting_metric_reader.py

```diff
@@ -84,7 +84,9 @@ def _create_periodic_reader(
         self, metrics, exporter, collect_wait=0, interval=60000
     ):
 
-        pmr = PeriodicExportingMetricReader(exporter, interval)
+        pmr = PeriodicExportingMetricReader(
```

Comment:
this change is unrelated to the PR

Reply:
It is actually related, since we are now adding another parameter this test case would fail without this change because it would use the value for interval where the temporality dictionary should go.

Reply:
Ah... yes.... that makes sense now. Thanks 👍